build.gradle

```diff
@@ -187,22 +187,5 @@ task ktlint(type: JavaExec, group: "verification") {
     args "components/**/*.kt" , "samples/**/*.kt", "!**/build", "buildSrc/**/*.kt"
 }
 
-task printModules {
-    doLast {
-        def modules = subprojects.collect { p -> [
-            name: p.path.toString(),
-            buildPath: p.buildDir.toString(),
-            shouldPublish: p.ext.shouldPublish
-        ] }
-        println "modules: " + groovy.json.JsonOutput.toJson(modules)
-    }
-}
-
-task printVersion {
-    doLast {
-        println("version: " + Config.componentsVersion)
-    }
-}
-
```

Comment:
Note that `printVersion` is used by `utomation/docs/generate_api-reference_release.sh` too.

Reply:
I've updated [`generate_api-reference_release.sh`](https://github.com/mozilla-mobile/android-components/pull/1694/files#diff-e570e00f7f642277c394446ff507c8deL9) as part of the PR :smile:

Reply:
Okay, I'm blind 😎